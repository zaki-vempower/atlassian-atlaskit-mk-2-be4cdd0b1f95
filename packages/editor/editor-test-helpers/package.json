{
  "name": "@atlaskit/editor-test-helpers",
  "version": "10.3.2",
  "private": true,
  "description": "Test Helpers for Atlassian Editor components",
  "license": "Apache-2.0",
  "atlaskit:src": "src/index.ts",
  "module": "dist/esm/index.js",
  "main": "dist/cjs/index.js",
  "types": "dist/cjs/index.d.ts",
  "author": "Atlassian Pty Ltd",
  "maintainers": [
    "Stanislav S"
  ],
  "atlaskit": {
    "internal": true,
    "name": "Editor Test Helpers",
    "team": "editor"
  },
  "repository": "https://bitbucket.org/atlassian/atlaskit-mk-2",
  "dependencies": {
    "@atlaskit/activity": "^1.0.1",
    "@atlaskit/adf-schema": "^4.3.1",
    "@atlaskit/analytics-gas-types": "^4.0.12",
    "@atlaskit/analytics-listeners": "^6.2.2",
    "@atlaskit/analytics-next": "^6.3.2",
    "@atlaskit/avatar": "^17.1.2",
    "@atlaskit/button": "^13.3.2",
    "@atlaskit/editor-common": "^43.0.0",
    "@atlaskit/editor-core": "^115.2.1",
    "@atlaskit/icon": "^19.0.11",
    "@atlaskit/media-core": "^31.0.1",
    "@atlaskit/media-test-helpers": "^25.2.4",
    "@atlaskit/smart-card": "^12.6.1",
    "@atlaskit/textarea": "^2.2.1",
    "@types/enzyme": "^3.1.15",
    "@types/react": "^16.8.0",
    "enzyme": "^3.10.0",
    "prosemirror-commands": "^1.0.7",
    "prosemirror-keymap": "^1.0.1",
    "prosemirror-model": "^1.7.0",
    "prosemirror-state": "^1.2.2",
    "prosemirror-tables": "0.9.5",
    "prosemirror-utils": "^0.9.6",
    "prosemirror-view": "^1.9.12",
    "react-intl": "^2.6.0"
  },
  "peerDependencies": {
    "react": "^16.8.0"
  },
  "devDependencies": {
    "@types/prosemirror-commands": "^1.0.1",
    "@types/prosemirror-keymap": "^1.0.1",
    "@types/prosemirror-model": "^1.7.2",
    "@types/prosemirror-state": "^1.2.0",
    "@types/prosemirror-view": "^1.9.0",
    "chai": "^4.1.2",
    "sinon": "^2.2.0"
  }
}
