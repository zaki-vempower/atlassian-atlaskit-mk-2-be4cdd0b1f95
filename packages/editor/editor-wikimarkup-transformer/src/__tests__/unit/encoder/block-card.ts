import { defaultSchema } from '@atlaskit/adf-schema';
import WikiMarkupTransformer from '../../../index';

import { doc, blockCard } from '@atlaskit/editor-test-helpers';

describe('ADF => WikiMarkup - BlockCard', () => {
  const transformer = new WikiMarkupTransformer();

  test('should convert blockcard node', () => {
    const node = doc(blockCard({ url: 'https://www.dropbox.com' })())(
      defaultSchema,
    );
    expect(transformer.encode(node)).toMatchSnapshot();
  });
});
