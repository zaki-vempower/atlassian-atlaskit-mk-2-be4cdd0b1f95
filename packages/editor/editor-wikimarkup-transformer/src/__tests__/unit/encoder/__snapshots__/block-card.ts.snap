// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ADF => WikiMarkup - BlockCard should convert blockcard node 1`] = `"[https://www.dropbox.com|https://www.dropbox.com|smart-card]"`;
